### ISSUE TITLE
request help: let custom plugin show in dashboard

### ISSUE THREAD

**Issue by thekingofworld** [apache/apisix-dashboard#646]
### Issue description
I have added a custom plugin, how to let the custom plugin show in dashboard?
![image](https://user-images.githubusercontent.com/8169665/97656163-c4dff300-1aa1-11eb-8b17-4f4e6f48ea06.png)

![image](https://user-images.githubusercontent.com/8169665/97656238-f35dce00-1aa1-11eb-977c-a6c1e5525752.png)

thx for help.

### Environment

* apisix version (cmd: `apisix version`): `2.0`
* OS: `mac os`

**Comment by juzhiyuan** [comment]
Hi, please follow those steps to have a check first:

1. Get all plugin list

```sh
$ curl $API_HOST/apisix/admin/plugins
```

2. Check if your `cgi-proxy` plugin is included in that list.

Looking forward to your feedback.

**Comment by juzhiyuan** [comment]
BTW, please use the latest dashboard 2.0 RC2 to have a test.

https://github.com/apache/apisix-dashboard/blob/v2.0/docs/deploy.md

**Comment by thekingofworld** [comment]
@juzhiyuan 
![image](https://user-images.githubusercontent.com/8169665/97659798-e0033080-1aaa-11eb-8780-47d109a019fd.png)

**Comment by juzhiyuan** [comment]
OK then, this error is returned from 2.0 RC2 may be, let's visit the dashboard then check this API:

![image](https://user-images.githubusercontent.com/2106987/97666230-2743ee00-1ab8-11eb-9567-e5059f35ba03.png)

**Comment by thekingofworld** [comment]
![image](https://user-images.githubusercontent.com/8169665/97669576-5f9afa80-1abf-11eb-9058-3d8b7ce7e40f.png)

**Comment by juzhiyuan** [comment]
Ok then this issue is not because of frontend, let’s cc @nic-chen @membphis to have more discussions with you :)

**Comment by nic-chen** [comment]
@thekingofworld 

need to re-generate json schema for your custom plugin.

here is how to use the tool to generate json schema:

https://github.com/apache/apisix-dashboard/blob/v2.0/docs/develop.md#sync-jsonschema

**Comment by juzhiyuan** [comment]
Oh! That’s it!!! The Dashboard needs this tool to regenerate plugin schemas, this trick tip should be removed and improve this part in the future.

**Comment by idbeta** [comment]
@thekingofworld We have reproduced this problem in the local environment, thank you for your feedback.

**Comment by membphis** [comment]
> @thekingofworld We have reproduced this problem in the local environment, thank you for your feedback.

do you find an easier way to reproduce this issue? or as the reporter?

**Comment by membphis** [comment]
I was confused with those labels. Welcome to split them into multiple different issues for easy tracking and fix.

![image](https://user-images.githubusercontent.com/6814606/98382709-81c7e600-2086-11eb-8bb6-ddf2037b3229.png)

**Comment by idbeta** [comment]
> > @thekingofworld We have reproduced this problem in the local environment, thank you for your feedback.
> 
> do you find an easier way to reproduce this issue? or as the reporter?

make a plugin with reference to example-plugin to reproduce the problem, update schema.json with `schema-sync.sh`  and it will report an error
```
--2020-11-06 19:47:50--  https://github.com/apache/apisix/archive/2.0.zip
...
...
200 OK
...
...
Archive:  2.0.zip
  End-of-central-directory signature not found.  Either this file is not
  a zipfile, or it constitutes one disk of a multi-part archive.  In the
  latter case the central directory and zipfile comment will be found on
  the last disk(s) of this archive.
unzip:  cannot find zipfile directory in one of 2.0.zip or
        2.0.zip.zip, and cannot find 2.0.zip.ZIP, period.
mv: rename ./apisix-2.0/apisix/* to ./api/build-tools/apisix/*: No such file or directory
lua: schema-sync.lua:89: module 'apisix.schema_def' not found:
	no field package.preload['apisix.schema_def']
	no file '/usr/local/share/lua/5.3/apisix/schema_def.lua'
	no file '/usr/local/share/lua/5.3/apisix/schema_def/init.lua'
	no file '/usr/local/lib/lua/5.3/apisix/schema_def.lua'
	no file '/usr/local/lib/lua/5.3/apisix/schema_def/init.lua'
	no file './apisix/schema_def.lua'
	no file './apisix/schema_def/init.lua'
	no file '/usr/local/lib/lua/5.3/apisix/schema_def.so'
	no file '/usr/local/lib/lua/5.3/loadall.so'
	no file './apisix/schema_def.so'
	no file '/usr/local/lib/lua/5.3/apisix.so'
	no file '/usr/local/lib/lua/5.3/loadall.so'
	no file './apisix.so'
stack traceback:
	[C]: in function 'require'
	schema-sync.lua:89: in main chunk
	[C]: in ?
sync success:
/Users/idbeta/1106/apisix-dashboard/api/conf/schema.json
```

**Comment by moonming** [comment]
@nic-chen is this a blocked issue？

**Comment by juzhiyuan** [comment]
This issue is reported 3 times at least, so it's a blocked issue IMO.

**Comment by membphis** [comment]
@idbeta please make a try with the latest branch `v2.0`, confirm if it works fine for you.

if it works fine, then we can close this issue.

**Comment by membphis** [comment]
@thekingofworld welcome to make a try too, wait for your response ^_^

**Comment by nic-chen** [comment]
fix by #676 #765
